packages/redux/src/exchanges/types/state.types.ts

```diff
@@ -0,0 +1,16 @@
+import type {
+  BlackoutError,
+  Exchange,
+  ExchangeBookRequest,
+  ExchangeFilters,
+} from '@farfetch/blackout-client';
+import type { CombinedState } from 'redux';
+import type { Nullable, StateWithResult } from '../../index.js';
+
+export type ExchangesState = CombinedState<{
+  error: BlackoutError | null;
```

Comment:
Since you are using Nullable already for `result`, you can use it for `error` as well.

```suggestion
  error: Nullable<BlackoutError>;
```